### PR TITLE
Consistent costs case

### PR DESCRIPTION
**Pull request by oliverguenther**
This changes english locale to be consistent with sentence case.
Moreover, this fixes the "Cost Types" admin entry.

**Please Note:** In the German locale, this changes 'Kostenarten' (plural) to 'Kostentypen', as is consistent with the singular german localized type (Kostentyp, cf. `label_cost_type_specific`).

Relevant work package: https://community.openproject.org/work_packages/20621

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -80,8 +79,8 @@ en:
     units: "Units"
     valid_from: "Valid from"
 
-  button_add_budget_item: "Add Planned Costs"
-  button_add_cost_object: "Add Budget"
+  button_add_budget_item: "Add planned Costs"
+  button_add_cost_object: "Add budget"
```
Comment (ulferts) [inline config/locales/en.yml:83]: @oliverguenther you missed one here -> `Costs`

File: config/locales/en.yml [hunk 2]
```diff
@@ -91,20 +90,20 @@ en:
 
   caption_booked_on_project: "Booked on Project"
   caption_default: "Default"
-  caption_default_rate_history_for: "Default Rate History for %{user}"
+  caption_default_rate_history_for: "Default rate history for %{user}"
   caption_labor: "Labor"
-  caption_labor_costs: "Actual Labor Costs"
+  caption_labor_costs: "Actual labor Costs"
```
Comment (ulferts) [inline config/locales/en.yml]: @oliverguenther you missed one here -> `Costs`

**Comment by ulferts** [comment]
@oliverguenther looks good. Thanks for doing the extra work. :+1:

**Comment by oliverguenther** [comment]
Missed + fixed three more than what you found. Thanks - my regex was off and didn't capture capitalized words at the end of the string.

**Pull request merged**